src/frontend/Komponenter/Behandling/BehandlingContainer.tsx

```diff
@@ -45,28 +48,39 @@ const BehandlingContainer: FC = () => {
     );
 };
 
+const BehandlingContent: FC<{
```

Comment:
Måtte splitte ut denne komponenten for å få til `useSetValgtPersonIdent`. 
Bedre navnforslag?

Reply:
Den bor jo inni `BehandlingContainer` - kan dét være et passende navn?

Reply:
Nei, den finnes fra før... 😂